Learning_test_automation/Spotkanie6_Zadania/CurrencyConversion_vBeta/Option/CurrencyPresentation.cs

```diff
@@ -41,27 +65,37 @@ private void PrintAvailableCurrencies()
             Console.WriteLine("- Czech crown: 3.\n");
         }
 
-        public void PrintAvailableBuyExchangeRates()
+        public void PrintAvailableListExchangeRatesValue()
         {
+            string caller = new StackTrace().GetFrame(1).GetMethod().Name;
+
             Console.WriteLine("\nHere are the values of currencies that we have prepared for you: ");
-            foreach (CurrencyDefault currency in Enum.GetValues(typeof(CurrencyDefault)))
+
+            foreach (var currency in currencies)
             {
-                if (currency == CurrencyDefault.Dollar) Console.WriteLine($"- {dollar.NameCurrency}: {dollar.BoughtCurrency} pln,");
-                else if (currency == CurrencyDefault.Euro) Console.WriteLine($"- {euro.NameCurrency}: {euro.BoughtCurrency} pln,");
-                else if (currency == CurrencyDefault.Czech_crown) Console.WriteLine($"- {czechCrown.NameCurrency}: {czechCrown.BoughtCurrency} pln.");
+                name = currency.NameCurrency.ToString();
```

Comment:
Po co? :p

Reply:
Bo dwa razy użyte. To tak nie działa, że jak coś występuje więcej niż raz to już należy Tworzyć zmienną.